go.mod

```
@@ -1,46 +1,60 @@
 module github.com/hashicorp/consul-aws
 
-go 1.14
+go 1.22.1
```

We got some CVE patches in `1.22.2` this week, so suggesting we bump to that
```suggestion
go 1.22.2
```